docs/api/1.x/quotas.rst

```diff
@@ -0,0 +1,91 @@
+.. _api-quotas:
+
+Quotas management
+#################
+
+When the built-in plugin ``kinto.plugins.quotas`` is enabled in
+configuration, it becomes possible to configure quotas for a bucket or
+a collection.
+
+Clients can check for the ``quotas`` capability in the
+:ref:`root URL endpoint <api-utilities>`.
+
+.. note::
+
+    In terms of performance, enabling this plugin generates two or
+    three additional queries on backends per request.
+
+* The bucket quota consider the bucket attributes, groups attributes,
```

Comment:
`considers`. I would also reword this, something like "A bucket's quota is a limit on the size of bucket attributes, group attributes, collection attributes, and record attributes."